rfc6962-bis.xml

```diff
@@ -835,6 +858,9 @@ signatures.
         <t>
           <spanx style="verb">root_hash</spanx> is the root of the Merkle Hash Tree.
         </t>
+        <t>
+          <spanx style="verb">extensions</spanx> are future extensions to STH v2. Currently, no extensions are specified. If an implementation sees an extension that it does not understand, it SHOULD ignore that extension. Furthermore, an implementation MAY choose to ignore any extension(s) that it does understand.
```

Comment:
Yes, this text matches the SCT text.

Maybe we should just remove the "Furthermore...that it does understand" sentence entirely?